.renovaterc.json

```diff
@@ -35,6 +35,11 @@
   "separateMajorMinor": true,
   "separateMinorPatch": false,
 
+  "packageRules": [{
+    "sourceUrlPrefixes": ["https://github.com/strongloop/loopback-next"],
```

Comment:
Shouldn't this be `https://github.com/strongloop/loopback4-example-shopping`?

Reply:
This rule is teaching RenovateBot about LB4 monorepo, we may want to contribute it to RenovateBot in the future. See existing monorepo definitions: https://renovatebot.com/docs/presets-monorepo/

With this rule enabled, RenovateBot will group updates of all packages published from loopback-next into a single commit/pull request. Without this rule, each `@loopback/*` package will be updated in a separate pull request.

If I changed the url prefix to loopback4-example-shopping as you suggested, then RenovateBot will group updates of packages from this monorepo. That does not make sense because these packages are local to this monorepo and dependency ranges are managed by lerna.

Reply:
Thanks for the explanation, sounds good to me 👍